koku/masu/api/urls.py

```diff
@@ -60,6 +68,14 @@
         name="get_one_manifest_file",
     ),
     path("gcp_invoice_monthly_cost/", gcp_invoice_monthly_cost, name="gcp_invoice_monthly_cost"),
+    path("db-performance/db-settings/", dbsettings, name="db_settings"),
+    path("db-performance/lock-info/", lockinfo, name="lock_info"),
+    path("db-performance/stat-activity/", stat_activity, name="conn_activity"),
+    path("db-performance/stat-statements/", stat_statements, name="stmt_stats"),
+    path("db-performance/db-version/", pg_engine_version, name="db_version"),
+    path("db-performance/cancel-backend-pid/", pg_cancel_backend, name="db_cancel_connection"),
```

Comment:
**issue(blocking): ** I think this endpoint should be purely for information gathering and we shouldn't allow canceling/terminating processes through this endpoint.
**suggestion(blocking): ** I think these should be removed, but I am open to discussion about this, maybe with more of the team on slack as well.